src/mobu/services/monkey.py

```diff
@@ -216,15 +187,22 @@ async def run_once(self) -> str | None:
         """
         self._state = MonkeyState.RUNNING
         error = None
-        try:
-            await self.business.run_once()
-            self._state = MonkeyState.FINISHED
-        except Exception as e:
-            msg = "Exception thrown while doing monkey business"
-            self._logger.exception(msg)
-            error = str(e)
-            self._state = MonkeyState.ERROR
-        return error
+        with sentry_sdk.isolation_scope():
+            sentry_sdk.set_user({"username": self._user.username})
+            sentry_sdk.set_tag("business", self.business.__class__.__name__)
+            try:
+                with start_transaction(
+                    name=self.__class__.__name__,
+                    op=self.__class__.__name__,
+                ):
```

Comment:
I think these were supposed to be `self.business.__class__.__name__`. Right now, isn't this always the literal string `Monkey`? (If I'm right, it would also be worth refactoring that value into a variable; it would make the code easier to read.